docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Contributing to GitOps Docs
+
+## Pre-requisites
+
+- Fork and clone the [operate-first/apps][apps-repo] repo
```

Comment:
"Fork and clone" could be a link to those instructions in the Community Handbook, but we don't quite have that page. Instead we have my page on using `o1-clone` instead (https://github.com/operate-first/community-handbook/blob/main/content/how-to-use-o1-clone-tool.md).

Do we have an existing page that tells how to fork and clone as we want them done? E.g. the instructions for the website?

(not a blocking review comment)

Reply:
> Do we have an existing page that tells how to fork and clone as we want them done? E.g. the instructions for the website? 

We don't, maybe we should.I think it's better to describe verbatim what we want them to do in this doc, then link to it from all other docs as part of a "Pre-requisites" section.

I'd suggest making an issue for this however as it's out of scope for this PR.